src/Runner.Worker/action_yaml.json

```diff
@@ -120,7 +120,22 @@
                 "hashFiles(1,255)"
             ],
             "sequence": {
-                "item-type": "any"
+                "item-type": "composite-step"
```

Comment:
do we have validation on the server that at least one step is required?

Reply:
For server side, if there are no "steps" token used, it handles it: https://github.com/ethanchewy/testing-actions/runs/898870893?check_suite_focus=true. If there are no steps within the "steps" token, the user will get an unexpected value from the server side: https://github.com/ethanchewy/testing-actions/runs/898862578?check_suite_focus=true